getting-started-zkapps.mdx

```diff
@@ -0,0 +1,232 @@
+---
+title: zkApps Getting Started
+sidebar_label: Getting Started
+hide_title: true
+description: How to get started writing zkApps (zero knowledge apps).
+keywords:
+  - zkapps
+  - mina protocol
+  - smart contracts
+  - zero-knowledge proofs
+  - off-chain execution
+  - typescript
+  - quickstart zkapps
+---
+
+:::info
+
+zkApp programmability is not yet available on the Mina Mainnet. You can get started now by deploying zkApps to the Berkeley Testnet.
+
+:::
+
+# zkApps Getting Started
```

Comment:
Has this file been duplicated into the project root folder?

Reply:
gulp, um, yes, let me move!